ansible/singleplatform-eng.users.yml

```diff
@@ -0,0 +1,3 @@
+- hosts: cluster
```

Comment:
Can you call it something more fitting for what it does rather than how it does it?  Something like infra-users.yml, for example?

Reply:
Hmm, there is already a playbook called `users.yml`. It appears to do a similar thing but uses roles called user_setup and user_propagate. I suppose the name needs to differentiate from that. Alternatively, we can replace them with singleplatform-eng.users role altogether.